README.md

```diff
@@ -39,6 +39,7 @@
 
 - **[Has independent classes]**
 - **[Use the "Decorator" design pattern]**
+- **[Has Associations]**
 
 
```

Comment:
- [ ] The issues page link in the README file is broken, Please make sure to fix it. This will make the README file more professional.